test/unit_tests/braket/devices/test_local_simulator.py

```diff
@@ -15,16 +15,26 @@
 from unittest.mock import Mock
 
 import pytest
+from pydantic import create_model  # This is temporary for defining properties below
```

Comment:
When do we expect this to use a schema model?

Reply:
It is a temporary solution. In fact, I just copy paste [this part](https://github.com/aws/amazon-braket-default-simulator-python/blob/9d994beca7e1bf7b23a9343c489d03c937a513a3/src/braket/analog_hamiltonian_simulator/rydberg/rydberg_simulator.py#L192) of the code to this unit test (yes it is a unit test instead of the integ test, for increasing the code coverage). 

To make this nicer, we will need to add device properties for the AHS simulators [here](https://github.com/aws/amazon-braket-schemas-python/tree/quera/src/braket/device_schema/simulators).